pallets/file-system/src/utils.rs

```diff
@@ -203,7 +324,10 @@ where
         // Remove storage request.
         <StorageRequests<T>>::remove(&location);
 
-        // TODO: initiate deletion request for SPs.
+        // Challenge BSP to force update its storage root to uninclude the file.
```

Comment:
```suggestion
        // Challenge BSPs who confirmed this storage request, to force update its storage root to uninclude the file.
```

Reply:
We should check first if this storage request has confirmed BSPs before going through the trouble of sending the priority challenge.